src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,81 @@
+package pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+
+public class HomePage extends BasePage {
+
+            public HomePage( WebDriver givenDriver) {
+                super(givenDriver);
+//                driver = givenDriver;//we can remove these ones since we have initialized them in basepage(inheritance using extends)
+//                wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+        }
+
+//        POM locators
+//        By firstPlaylist = By.cssSelector(".playlist:nth-child(3)");
+//        By playlistInputField = By.cssSelector("input[name='name']");
+//        By userAvatarIcon = By.cssSelector("img.avatar");
+
+//        Page Factory locators
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.